src/python/interpret/ext/blackbox/__init__.py

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) 2019 Microsoft Corporation
+# Distributed under the MIT software license
+
+import logging
+import sys
+
+from interpret.ext.extension_utils import load_class_extensions
+
+module_logger = logging.getLogger(__name__)
+
+BLACKBOX_EXTENSION_KEY = "interpret_ext_blackbox"
+
+
+def _is_valid_blackbox_explainer(proposed_blackbox_explainer):
+    return True
```

Comment:
The idea here is we build this out as we figure out exactly what our parameters for blackbox explainers are?

Reply:
Maybe, we noticed an issue with validation and required parameters.

For now validation is converging to has explain_local or explain_global defined.

We are still figuring out if it is possible to instantiate the class with default args and require a _get_validation_init if the default instantiate would not work.